node/commons/compatibility-layer.ts

```diff
@@ -378,7 +378,7 @@ const convertSKU = (
  * @param {OrderBy} orderBy VTEX OrderBy.
  * @returns {string} Biggy's sort.
  */
-export const convertOrderBy = (orderBy?: string): string => {
+export const convertOrderBy = (orderBy?: string | undefined | null): string => {
```

Comment:
```suggestion
export const convertOrderBy = (orderBy?: string | null): string => {
```
the parameter is already optional